libs/zedUpload/azureutil/azure.go

```diff
@@ -29,14 +29,11 @@ const (
 	parallelism          = 16
 )
 
-// UpdateStats contains the information for the progress of an update
-type UpdateStats struct {
-	Size          int64    // complete size to upload/download
-	Asize         int64    // current size uploaded/downloaded
+// Resp contains the information for the response of an update
+type Resp struct {
 	List          []string //list of images at given path
-	Error         error
-	BodyLength    int   // Body legth in http response
-	ContentLength int64 // Content length in http response
+	BodyLength    int      // Body legth in http response
+	ContentLength int64    // Content length in http response
 
 	DoneParts types.DownloadedParts //downloaded parts
 }
```

Comment:
Cannot find any usage of this structure, do we need it?

Reply:
You are right, it is not. I was very conservative with removing things, but not only is it not used here, I checked out `master` branch and went through it carefully, it wasn't used there at all. Removed.